### ISSUE TITLE
Linux Build VS Quick Installation

### ISSUE THREAD

**Issue by tkddnjs98** [carla-simulator/carla#6365]
I don't know what is big difference between building on linux and quick installation when downloading carla.
I'm downloading carla because I have to test scenarios using Ros.
I found that I got too much errors when building on Carla.
Is there many system restriceted when using quick installation?

**Comment by Gaochengzhi** [comment]
Yes, you can't modify the whole scene using Unreal Engine editor if you don't build from source.
